Services/ItemService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Services.DTO;
+using Services.Interfaces;
+
+namespace Services
+{
+    public class ItemService : IItemService
+    {
+        public void Save(ItemDto dto)
+        {
```

Comment:
Need to map ItemDTO into an Item domain model object

Reply:
updated